qa/tasks/cephfs/test_volumes.py

```diff
@@ -9122,3 +9122,48 @@ def test_volumes_module_finisher_thread(self):
 
         # verify trash dir is clean
         self._wait_for_trash_empty()
+
+class TestEmptyStringForCreates(CephFSTestCase):
+    CLIENTS_REQUIRED = 1
+    MDSS_REQUIRED = 1
+
+    def setUp(self):
+        super().setUp()
+        result = json.loads(self.run_ceph_cmd("fs", "volume", "ls"))
+        self.assertTrue(len(result) > 0)
+        self.volname = result[0]['name']
+
+    def tearDown(self):
+        # clean up
+        super().tearDown()
+
+    def test_empty_name_string_for_subvolumegroup_name(self):
+        """
+        To test that an empty string is unacceptable for a subvolumegroup name
+        """
+        with self.assertRaises(CommandFailedError):
+            self.run_ceph_cmd("fs", "subvolumegroup", "create", self.volname, "")
+
+        with self.assertRaises(CommandFailedError):
+            self.run_ceph_cmd("fs", "subvolumegroup", "create", self.volname, "''")
```

Comment:
```suggestion
            self.run_ceph_cmd("fs", "subvolumegroup", "create", self.volname, "")
```

Reply:
The `"''"` is needed to force passing an empty string on the command-line without which the empty string (`""`) causes an empty string to be appended to the entire command-line causing an undesired NO-OP.